marketplace/gatsby-preview/src/AppConfig.js

```diff
@@ -57,7 +57,7 @@ export default class AppConfig extends React.Component {
         webhookUrl: params.webhookUrl || '',
         authToken: params.authToken || ''
       };
-    });
+    }, app.setReady);
```

Comment:
here Jakub had argued before we should better use `() => app.setReady()` to avoid inadvertently passing down any arguments just in case